### PR TITLE
Usman Jamshed post

### PR DESCRIPTION
**Pull request by vinnyasa**
- [ ] Header with name
- [ ] Answer to questions in blog form
- [ ] Image

### CODE REVIEW

File: _posts/2021-08-09-Usman-Jamshed.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+</center>
+
+
+# Usman Jamshed
+Usman was born in Brooklyn, NY.
+He's been living in Toronto since he was 8 years old.
+
+## Pets
+He has a cat pet called sage like the color :)
+When he was growing up his family wanted low commitment animals, so when he had about 10 boggies.
```
Comment (ujamshed) [inline _posts/2021-08-09-Usman-Jamshed.md]: Boggies should be budgies

File: _posts/2021-08-09-Usman-Jamshed.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+He's been living in Toronto since he was 8 years old.
+
+## Pets
+He has a cat pet called sage like the color :)
+When he was growing up his family wanted low commitment animals, so when he had about 10 boggies. 
+Some flew away and some died :(. 
+Then they got a cat :).
+
+## Food
+His fav: Slice of pizza, a plain thin cheese pizza. He really prefers pizza cooked in top stone oven. High fat low mouiture, it does not split as easily, it is very watery. No garnish just plain cheese.
```
Comment (ujamshed) [inline _posts/2021-08-09-Usman-Jamshed.md]: mouiture should be moisture, and it should be less watery

File: _posts/2021-08-09-Usman-Jamshed.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Interests
+Learning new things is one of his main iterests. 
+
+He likes soccer, tennis, and swimming. He competed at high-school level. 
+His main stroke is free-style and you will hardly even see him backsrtoking.
+
+Also, He likes eating the fresh tomatoes his father grows. His father grows heirloom tomatoes, zuchinis, and eggplants on their back garden. He mentioned the tomatoes.
+
+He considers himself as outdoorsy. He has never gone camping or fishing, but would like to go in that type of adventure someday. He likes going hiking around the area or even during travels to different type rated trails. He remembers the Vancouver Jokffre Lake trail as one of the most challenging ones his done. Mainly because he was fasting for Ramadan and it was tough to make to the end of the 6 hour hike.
```
Comment (ujamshed) [inline _posts/2021-08-09-Usman-Jamshed.md]: Should be Joffre

File: _posts/2021-08-09-Usman-Jamshed.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+I found three things we have in common:
+* We both love wood oven or stone top oven thin cheese pizzas.
+* We both love animals.
+* We both really like hiking and go on hiking trips. 
+
+## Interests
+Learning new things is one of his main iterests. 
+
+He likes soccer, tennis, and swimming. He competed at high-school level. 
+His main stroke is free-style and you will hardly even see him backsrtoking.
```
Comment (ujamshed) [inline _posts/2021-08-09-Usman-Jamshed.md]: Should be backstroke

**Review by ujamshed — CHANGES_REQUESTED** [review]
Hi, there are a couple spelling errors but other than that looks great!

File: _posts/2021-08-09-Usman-Jamshed.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+```
```
Comment (janash) [inline _posts/2021-08-09-Usman-Jamshed.md]:
```suggestion

```

File: _posts/2021-08-09-Usman-Jamshed.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+---
+layout: post
+title: "Team2 Interview: Usman Jamshed"
+author: Vinny Harris-Riviello
+---
+```
```
Comment (janash) [inline _posts/2021-08-09-Usman-Jamshed.md]:
```suggestion

```

File: _posts/2021-08-09-Usman-Jamshed.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+<center>
+
+<img src = '{{ "/images/msse-logo-bg.svg" | relative_url }}'>
```
Comment (janash) [inline _posts/2021-08-09-Usman-Jamshed.md]:
```suggestion
<img src = '{{ "/images/sage.JPG" | relative_url }}'>  
```

**Review by janash — CHANGES_REQUESTED** [review]
Hey Vinny and Usman, very nice blog post, I really enjoyed reading it. I have a few feedback items on the format that should be addressed.

I've suggested four changes. 

1. For the first two changes, you have bacticks surrounding the page header. This may cause that text to be rendered literally rather than to be picked up by the template. 
2.  I noticed an asterisks (`*`)  at the end of the figure caption. This is making the rest of the article appear bold, so it would be best to remove it.
3. Make sure to update the figure URL. You've uploaded a picture of Sage, but the URL is still pointing at the MSSE logo. Be sure to change the file name in the URL.

Thanks! Hope you don't mind me adding a review as well.

**Review by janash — APPROVED** [review]
I'm approving in case the edits go through when I'm not available

File: _posts/2021-08-09-Usman-Jamshed.md (hunk 2 above)
Comment (vinnyasa) [reply _posts/2021-08-09-Usman-Jamshed.md]: I actually did not mean to put that comment so I am deleting it

File: _posts/2021-08-09-Usman-Jamshed.md (hunk 3 above)
Comment (vinnyasa) [reply _posts/2021-08-09-Usman-Jamshed.md]: done

File: _posts/2021-08-09-Usman-Jamshed.md (hunk 5 above)
Comment (vinnyasa) [reply _posts/2021-08-09-Usman-Jamshed.md]: Thanks

**Comment by vinnyasa** [comment]
Thanks for your reviews @ujamshed and @janash

**Review by ujamshed — APPROVED** [review]

**Pull request merged**